beacon-chain/blockchain/kzg/kzg.go

```diff
@@ -0,0 +1,138 @@
+package kzg
```

Comment:
Why is this package located in `blockchain` instead of a place like `consensus-types`? Curious if there’s any rationale or tradeoff behind that decision

Reply:
The new `kzg.go` file has been added here because the `kzg` package was located here.
Seems Potuz created this package [here](https://github.com/OffchainLabs/prysm/commit/57b8da08d1ee63e29212ab5094840c89eab7d186#diff-a39422a3ab7fad4b60b82583db2ca9e7e227487c56b1f678f0f1e5a2a636fb9c).

I'm not opposed to move it elsewhere. I don't have any strong opinion on that.